src/components/Button.tsx

```diff
@@ -31,7 +31,7 @@ const buttonClasses = (
   variant: ButtonVariant = "base"
 ) =>
   clsxm(
-    "inline-grid grid-cols-1 grid-rows-1 items-center justify-center rounded-md px-5 py-3 disabled:opacity-75",
+    "inline-grid grid-cols-1 grid-rows-1 font-medium items-center justify-center rounded px-5 py-3 disabled:opacity-75",
```

Comment:
Less rounded corners make the buttons appear natural inside a box that is `rounded-md`. Increasing the font weight because it is medium in most places (let's be consistent).